CONTRIBUTING.md

```diff
@@ -163,25 +175,30 @@ implementation 'com.meilisearch.sdk:meilisearch-java:X.X.X'
 
 Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/meilisearch-java/releases).
 
-#### Sign your files and upload to Maven Repository <!-- omit in TOC -->
+#### Sign your files and upload to Maven Repository manually <!-- omit in TOC -->
 
-1. Prepare the environment by filling the `gradle.properties` file with all the credentials:
+1. Prepare the environment by filling the environment variables needed with all the credentials:
```

Comment:
Are this variables stored at `gradle.properties`? We can specify this on that step

Reply:
Oh, I see this

> gradle.properties has been deleted and we now use environment variables

Is there a reason for removing `gradle.properties`?

Reply:
I don't store any variables in `gradle.properties` anymore because they are used as environment variables or in the command by the publishing workflow.
That's why I am wondering if we still need those steps in the CONTRIBUTING.MD. If so, we might have a specific manual process (using `gradle.properties`).